.gitignore

```diff
@@ -177,6 +177,8 @@ tests/data/schema.gpickle
 # Created during testting
 Example*
 manifests/*
+test_cache_dir
```

Comment:
when will test_cache_dir get created? If it is created by test, it should be cleaned up after the test is done?

Reply:
This now gets deleted in the `login` test

Reply:
should this line be removed?

Reply:
removed